packages/vite-plugin-kit-routes/src/lib/ROUTES.ts

```diff
@@ -61,8 +61,8 @@ const PAGES = {
     params.lang = params.lang ?? 'fr'
     params.id = params.id ?? 'Vienna'
     return `${params?.lang ? `/${params?.lang}` : ''}/site/${params.id}${appendSp({
-      limit: params?.limit,
-      demo: params?.demo,
+      limit: StringOrUndefined(params.limit),
+      demo: StringOrUndefined(params.demo),
```

Comment:
It should be 
```ts
StringOrUndefined(params?.limit),
``` 
no?

Reply:
`params` is definitively set in line 60. I struggled so long to get the `?.` operator to bet used in the correct places :sweat_smile: